opennlp-tools/src/main/java/opennlp/tools/entitylinker/LinkedSpan.java

```diff
@@ -134,9 +134,8 @@ public boolean equals(Object obj) {
       return false;
     }
     final LinkedSpan<T> other = (LinkedSpan<T>) obj;
-    if (!Objects.equals(this.linkedEntries, other.linkedEntries)) {
-      return false;
-    }
-    return this.sentenceid == other.sentenceid && Objects.equals(this.searchTerm, other.searchTerm);
+    return Objects.equals(this.linkedEntries, other.linkedEntries) &&

```

Comment:
Can you remove this? OPENNLP-922 changes that in one style across all equals and hashCode implementations.